### PR TITLE
Update settargeting arguments

### PR DESCRIPTION
**Pull request by alanhk89**
Can refer to this issue - https://github.com/jaanauati/react-dfp/issues/26

@jaanauati When we call `DFPManager.setTargetingArguments()`, it will also update the targeting arguments from googletag one.

### CODE REVIEW

File: js/manager.js [hunk 1]
```diff
@@ -11,6 +11,14 @@ const globalTargetingArguments = {};
 const DFPManager = Object.assign(new EventEmitter().setMaxListeners(0), {
   setTargetingArguments(data) {
     Object.assign(globalTargetingArguments, data);
+    this.getGoogletag().then((googletag) => {
+      googletag.cmd.push(() => {
+        pubadsService = googletag.pubads();
```
Comment (jaanauati) [inline js/manager.js]: can u please declare a local var:
``` const pubadsService = googletag.pubads(); ```
i know, out of this pr's scope :-), can u please make the same change to the line *42* and  delete the variable declared in line 7 ?.

**Comment by jaanauati** [comment]
This makes sense, nice catch, i will test it on my local as soon as you push the changes.

Thanks!.

File: js/manager.js (hunk 1 above)
Comment (alanhk89) [inline js/manager.js]: Sure, no problem.
And done.

**Comment by alanhk89** [comment]
I have pushed the changes base on your request.

Thanks!

File: js/manager.js [hunk 2]
```diff
@@ -4,13 +4,20 @@ import * as Utils from './utils';
...
 let googleGPTScriptLoadPromise = null;
 const registeredSlots = {};
-let pubadsService = null;
 let managerAlreadyInitialized = false;
 const globalTargetingArguments = {};
 
 const DFPManager = Object.assign(new EventEmitter().setMaxListeners(0), {
   setTargetingArguments(data) {
     Object.assign(globalTargetingArguments, data);
+    this.getGoogletag().then((googletag) => {
```
Comment (jaanauati) [inline js/manager.js]: tests well on my local, I only found a little issue for  a pattern like the following, which will end up creating a double assignment:
```
...
DFPManager.setTargetingArguments({ foo: 'bar' });
DFPManager.load();
...
```
debugger:
```
Setting targeting attribute foo with value bar for service publisher_ads
Setting targeting attribute foo with value bar for service publisher_ads
```
We can fix this just surrounding this block with the following:
```
if (loadAlreadyCalled === true) {
  this.getGoogletag().then((googletag) => {
    googletag.cmd.push(() => {
      ..
    }
  });
}
```

Thanks!

Comment (jaanauati) [inline js/manager.js]: updated comment, please use the flag *managerAlreadyInitialized*

Comment (alanhk89) [inline js/manager.js]: Updated base on your comment, please help to check again.

Thanks

**Comment by alanhk89** [comment]
Updated base on your comment, please help to check again.

And do I need to update both `manager.js` in folder `lib` and `js`?

Thanks

**Comment by jaanauati** [comment]
thumbs-up

**Comment by jaanauati** [comment]
I just published a new version with your fix. Thank you!.

**Pull request merged**